### PR TITLE
Add project board

### PR DESCRIPTION
**Pull request by christopher-hakkaart**

### CODE REVIEW

File: src/content/events/2023/hackathon-november-2023.mdx [hunk 1]
```diff
@@ -115,11 +115,14 @@ Ensure you have read/signed up/joined/installed the following resources before t
 
 # Groups
 
-To keep work and discussion during the hackathon manageable, we will organise everyone into groups.
-You are free to change groups as you wish during the hackathon.
+This nf-core hackathon does not have formalized projects.
+Instead, we have GitHub issues that are categorized into groups.
+The groups are used to help organise issues, discussion, and reporting during the hackathon.
+You are not restricted to staying in any single group are free to change groups as you wish during the hackathon.
```
Comment (mashehu) [inline src/content/events/2023/hackathon-november-2023.mdx:121]:
```suggestion
You are not restricted to staying in any single group and are free to change groups as you wish during the hackathon.
```

File: src/content/events/2023/hackathon-november-2023.mdx [hunk 2]
```diff
@@ -115,11 +115,14 @@ Ensure you have read/signed up/joined/installed the following resources before t
...
+You are welcome to add your own issues to the hackathon so others can join you.
```
Comment (mashehu) [inline src/content/events/2023/hackathon-november-2023.mdx:122]:
```suggestion
```
Moved down to next paragraph

File: src/content/events/2023/hackathon-november-2023.mdx [hunk 3]
```diff
@@ -115,11 +115,14 @@ Ensure you have read/signed up/joined/installed the following resources before t
...
 
-We will coordinate our work and the issues we are working on using a single GitHub project board (we'll post a link here closer to the time).
-There will be a separate tab for each group where will have collated a list of relevant issues.
+We will coordinate our work and the issues we are working on using a single [GitHub project board](https://github.com/orgs/nf-core/projects/56).
+There will be a separate tab for each group where will collate relevant issues.
```
Comment (mashehu) [inline src/content/events/2023/hackathon-november-2023.mdx:125]:
```suggestion
We will coordinate our work and the issues we are working on using a single [GitHub project board](https://github.com/orgs/nf-core/projects/56).
There will be a separate tab for each group where will collate relevant issues.
You are welcome to add your own issues to the project board so others can join you.
```

**Comment by christopher-hakkaart** [comment]
Thanks @mashehu

**Review by ewels — APPROVED** [review]

**Pull request merged**